apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -200,16 +200,20 @@ export default class AnimationPickerBody extends React.Component {
   }
 
   render() {
+    let appType;
     let assetType;
     switch (this.props.pickerType) {
       case PICKER_TYPE.spritelab:
         assetType = msg.costumeMode();
+        appType = 'spritelab';
         break;
       case PICKER_TYPE.gamelab:
         assetType = msg.animationMode();
+        appType = 'gamelab';
         break;
       case PICKER_TYPE.backgrounds:
         assetType = msg.backgroundMode();
+        appType = 'spritelab';
```

Comment:
Not sure we care about this level of detail, but where would a lab that subclasses spritelab (eg, poetry) fit in here?

Reply:
We're adding anaytics to decide whether to moderate all custom images (include App/Game Lab) or just images uploaded in labs geared for K-5 (Sprite lab and subtypes). So I think probably works best for that purpose. Thanks!